alerting/build.gradle

```diff
@@ -164,6 +166,151 @@ integTest {
     if (System.getProperty("test.debug") != null) {
         jvmArgs '-agentlib:jdwp=transport=dt_socket,server=n,suspend=y,address=8000'
     }
+
+    if (System.getProperty("tests.rest.bwcsuite") == null) {
+        filter {
+            excludeTestsMatching "org.opensearch.alerting.bwc.*IT"
+        }
+    }
+}
+
+String bwcVersion = "1.13.1.0"
```

Comment:
Do we want to add all the older versions (and add the zips) and test BWC against all of those? This would mean we also test against OS 1.0 and OS 1.1 since this is on OS 1.2.

Reply:
Answered a similar question to one of @skkosuri-amzn's comments. We need to check OpenSearch core's BWC test matrix for older versions so we can have parity with that but for now I'm at least including against ODFE `1.13.1` since it's a common upgrade path which spans OpenDistro and OpenSearch.